templates/python/setup.mustache

```diff
@@ -22,7 +22,7 @@ REQUIRES = [
     "six>=1.10",
     "urllib3>=1.23",
     "Authlib>=1.0.0,<1.1.0",
-    "requests<=2.29.0"
+    "requests>=2.31.0"
```

Comment:
Could you apply this patch [1] and then run `make python`, and commit the changes?

```patch
diff --git a/templates/python/setup.mustache b/templates/python/setup.mustache
index 8c06d85..c6b490c 100644
--- a/templates/python/setup.mustache
+++ b/templates/python/setup.mustache
@@ -22,7 +22,7 @@ REQUIRES = [
     "six>=1.10",
     "urllib3>=1.23",
     "Authlib>=1.0.0,<1.1.0",
-    "requests>=2.31.0"
+    "requests >= 2.0.0, <= 3.0.0"
 ]
 
 {{#asyncio}}
```

Reply:
Used `<3.0` for the upper bound, but otherwise as requested.

Reply:
Sorry, one more try on my end, lost my change somehow. Also I now see the `requirements.txt`, so I'll use that exactly.

Reply:
No worries! Yeah, there's the templating step which isn't immediately obvious!